### ISSUE TITLE
[Feature Request]: Default locale (formatting data - Times, dates etc.)  is US.

### ISSUE THREAD

**Issue by aknight1-uva** [jasp-stats/jasp-issues#39]
Currently, JASP starts with locale set to U.S. My guess this was done as a "fix" for issue #549.
This "breaks" all the locale-sensitive string manipulation done by QT.
The fix is to use QString instead of std::string for manipulating user input and user presentation (and while that is being done, replace std::string for QString everywhere where there is no real need for std::string).
That done we can allow the JASP to use the system locale - giving dd-mm-yyyy format dates and comma as decimal separator, for NL users, dd/mm/yyyy and dot as dec. sep. for UK and mm/dd/yyyy and dot for US.
Putting application locale as an option in the config files (default: 'system') would allow users to choose what is appropriate for them.
Note: For those not familiar with QT locales see: https://wiki.qt.io/Locales

**Comment by JorisGoosen** [comment]
@boutinb do we do this now?

**Comment by tomtomme** [comment]
no, it is still an open issue. I can switch to german for the UI, but result tables still show for example 2.2 instead of 2,2.

**Comment by JorisGoosen** [comment]
I've moved this into 0.15 because I think it is hightime we start working properly with locales. Especially now we are starting to support more languages.

**Comment by tomtomme** [comment]
would still be a cool feature ;)

**Comment by JorisGoosen** [comment]
Sure is!
Its definitely something we will do, also together with #30 or perhaps https://github.com/jasp-stats/INTERNAL-jasp/issues/1744 and the rest

**Comment by tomtomme** [comment]
Cool :)
I cannot see those internal issues however :(

**Comment by JorisGoosen** [comment]
well its for  monetary output column, and I would extend that with a monetary datacolumn too.
But that would be locale independent in any case.
